modules/issue_tracker/README.md

```diff
@@ -1,11 +1,11 @@
 # Issue Tracker
 
 ## Purpose
-The Issues Module allows users to track issues they have with data, or with their LORIS instance itself. A form with pre-defined fields is provided for users to submit issues, and a filter-form gives a sortable and filterable table view of issues viewable by the user.
+The Issues Module allows users to track issues they have with data, or with their LORIS instance itself. A form with pre-defined fields is provided for users to submit issues, upload attachments and a filter-form gives a sortable and filterable table view of issues viewable by the user.
 
 ## Permissions
-- `issue_tracker_reporter` permission allows adding an issue, editing issues created by the user, and commenting on all issues for the site. 
-- `issue_tracker_developer` permission allows to do the same, as well as closing an issue or editing any field of a submitted issue for the site. 
+- `issue_tracker_reporter` permission allows adding an issue, editing issues created by the user, download issue attachments and commenting on all issues for the site. 
+- `issue_tracker_developer` permission allows to do the same, as well as closing an issue, editing any field of a submitted issue for the site and deleting issue attachments. 
 - `view_all_sites` permission allows a user to view issues relevant to data from other sites. 
```

Comment:
This permission doesn't seem to actually exist.

Reply:
@johnsaigle the permission to check if the user can delete are there. I just added $user->hasPermission('issue_tracker') to _hasAccess.

Reply:
This one says view_all_sites though and that's not in the code.

Reply:
@johnsaigle I'm not sure if view_all_sites is supposed to be able to access the issues or just see the issues that have been made. @jesscall do you know what view_all_sites should have access to when it comes to attachments or even viewing the issue?

Reply:
I don't think `view_all_sites` exists as a permission in LORIS so you'll have to add it to your SQL files if you want to use it. If you do it should also be something like `issue_tracker_view_all_sites` so it's more clear

Reply:
@ridz1208 @driusan should the _hasAccess inside the issue_tracker/php/endpoints/attachment.class.inc contain hasAnyPermission for issue_tracker and view_all_sites or doesn't issue_tracker solve that.